app/javascript/components/components/Footer.test.js

```diff
@@ -0,0 +1,14 @@
+// import React from 'react'
+// import Footer from './Footer'
+// import Enzyme, { shallow } from 'enzyme'
+// import Adapter from 'enzyme-adapter-react-16'
+
+// Enzyme.configure({ adapter: new Adapter() })
+
+// describe('When Header renders', () => {
+//   it('displays 2 NavLink', () => {
+//     const header = shallow(<Header />)
+//     const headerNav = header.find('NavLink')
+//     expect(headerNav.length).toEqual(2)
+//   })
+// })
```

Comment:
This commented out test should be breaking your testing suite. Please either delete this file or finish the testing for this component before the PR.